### PR TITLE
WEBUI-572: update docker action to v2

### PR DESCRIPTION
**Pull request by richardsd**

### CODE REVIEW

File: .github/workflows/preview.yaml [hunk 1]
```diff
@@ -62,27 +62,34 @@ jobs:
...
+
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
       with:
-        path: server
         registry: ${{ env.DOCKER_REGISTRY }}
         username: _json_key
         password: ${{ secrets.GKE_SA_KEY }}
+
+    - name: Build and push server image v2
```
Comment (Gabez0r) [inline .github/workflows/preview.yaml]: Do we need to keep this "v2" in the name?

File: .github/workflows/preview.yaml [hunk 2]
```diff
@@ -62,27 +62,34 @@ jobs:
...
+      uses: docker/build-push-action@v2
+      with:
+        context: server
+        pull: true
         push: true
-        repository: ${{ secrets.GKE_PROJECT }}/webui/nuxeo-web-ui/server
-        tags: ${{ env.VERSION }}
+        file: server/Dockerfile
+        tags: ${{ env.DOCKER_REGISTRY }}/${{ secrets.GKE_PROJECT }}/webui/nuxeo-web-ui/server:${{ env.VERSION }}
```
Comment (Gabez0r) [inline .github/workflows/preview.yaml]: Isn't this the same as?
```suggestion
        tags: ${{ env.DOCKER_REGISTRY }}/${{ env. DOCKER_IMAGE }}/server:${{ env.VERSION }}
```

File: .github/workflows/preview.yaml [hunk 3]
```diff
@@ -62,27 +62,34 @@ jobs:
...
 
-    - name: Build and push image
-      uses: docker/build-push-action@v1
+    - name: Build and push image v2
```
Comment (Gabez0r) [inline .github/workflows/preview.yaml]: Do we need to have this v2 here?

File: .github/workflows/preview.yaml (hunk 1 above)
Comment (richardsd) [reply .github/workflows/preview.yaml]: Nope, forgot to remove the v2 😬

File: .github/workflows/preview.yaml (hunk 3 above)
Comment (richardsd) [reply .github/workflows/preview.yaml]: Same as the other, to remove.

File: .github/workflows/preview.yaml (hunk 2 above)
Comment (richardsd) [reply .github/workflows/preview.yaml]: It is in fact. I'll update this one too. Thanks

**Review by Gabez0r — APPROVED** [review]

**Review by semisse — APPROVED** [review]

**Pull request merged**